File: java/Elektronik.java
// Elektronik.java
public class Elektronik extends Produk {
    String deskripsi;
    String kategori;

    // Constructor
    public Elektronik(int id, String namaProduk, String merk, float harga, int stok, String gambarPath, String deskripsi, String kategori) {
        super(id, namaProduk, merk, harga, stok, gambarPath);
        this.deskripsi = deskripsi;
        this.kategori = kategori;
    }

    // Override method display() untuk menampilkan lebih banyak informasi
    @Override
    public void display() {
        super.display(); // Menampilkan data dari Produk
        System.out.printf("Deskripsi: %-50s Kategori: %-20s\n", deskripsi, kategori);
    }
}

File: php/kode program/Elektronik.php
<?php
require_once 'Produk.php';

class Elektronik extends Produk {
    public $Kategori_produk;
    public $Deskripsi_produk;

    public function __construct($id, $Nama_produk, $Harga, $Stok, $GambarPath, $Kategori_produk, $Deskripsi_produk) {
        parent::__construct($id, $Nama_produk, $Harga, $Stok, $GambarPath);
        $this->Kategori_produk = $Kategori_produk;
        $this->Deskripsi_produk = $Deskripsi_produk;
    }

    public function display() {
        parent::display();
        printf(" %-20s %-50s", 
            $this->Kategori_produk, 
            $this->Deskripsi_produk
        );
    }
}
?>